doc: add ReleaseHOWTO document (#43)
harvester/node-disk-manager: ReleaseHOWTO.md

# How to generate a new release to harvester

1. Tag the version on [harvester/node-disk-manager](https://github.com/harvester/node-disk-manager)
 repository.

    (Use the github release action. Draft a new release with the related version and its commit.)

2. Bump the new version on [harvester/charts](https://github.com/harvester/charts)

>     - Working on the the `release` branch

Reviewer: If there are chart changes, we should work in the master first and backport to release.

Author: done, please check it again.

    - Now, the `version` and `apiVersion` are synced on NDM. That means no matter if you want to change the content of chart or update image you need to bump these two versions.

    (Check the [here](https://github.com/harvester/charts/tree/master/charts/harvester-node-disk-manager) would help to know more detail.)

> 3. Final, bump the version on [harvester/harvester](https://github.com/harvester/charts)

Reviewer: ```suggestion
3. Bump the version on [harvester/harvester](https://github.com/harvester/charts)
```

    - Working on [deploy/charts/harvester](https://github.com/harvester/harvester/tree/master/deploy/charts/harvester)

>     - The node-disk-manager is the dependency chart of harvester. You need to bump the related version (as above) and update the chart. (TIPS Command: `helm dependency`)

Reviewer: ```suggestion
    - The node-disk-manager is a Harvester dependency chart. You need to bump the related version (as above) and update the chart. (Command: `helm dependency`)
```
